build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -171,7 +171,9 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         {
             var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
             var configuration = c.BuildContext.Get<string>("Configuration");
-            string rid = c.BuildContext.Get<string>("TargetRID");
+            string rid = DotNetCli.Stage0.GetRuntimeId();
```

Comment:
>Which other products are you referring to Ian?

The package RIDs use the TARGETRID variable. If we remove that line of code you linked then the packages we would build would be e.g. `runtime.win10-x64.Microsoft.NETCore.App.2.0.0-*` which they never have been before.

>@jyoungyun is correct that this change will break cross-arch builds.

Are cross-arch builds something new since my original changes in August were done? This PR is reverting the behavior to how it used to be before my fix accidentally touched it, so it shouldn't put us in any worse of a state than we were before my changes.